serial_communication/web/app.py

```diff
@@ -153,21 +164,25 @@ def one_time_task():
                         str(random.randint(10000, 99999)))
                     BOOT_MESSAGE_SEND = True
                 time.sleep(5)
-            say_to_serial("sms sending?")
-            time.sleep(15)
-            send_ngrok_link()
+                say_to_serial("sms sending?")
+                time.sleep(15)
+                send_ngrok_link()
     except Exception as ott: # pylint: disable=broad-except
         exception_logger("one_time_task", ott)
 
 if __name__ == '__main__':
     try:
         # lsof -i :6677 #to know which process is using this port
         current_time = datetime.now()
+        initialize_port()
+        flash()
+        beep()
```

Comment:
I don't thing they work fine do they?

Reply:
yes, you are right, I just currently make our app able to communicate with watcher.